osbs/http.py

```diff
@@ -43,7 +43,7 @@ def decoded_json(iterable):
         if isinstance(line, str):
             yield line
         else:
-            yield line.decode(requests.utils.guess_json_utf(line))
+            yield line.encode(requests.utils.guess_json_utf(line))
```

Comment:
I'm wondering if we need the encode at all...
`requests.utils.guess_json_utf` will return the current encoding for the line.
Why encode it again?